src/cljs/org/broadinstitute/firecloud_ui/common.cljs

```diff
@@ -97,10 +97,12 @@
 
 
 (defn compute-status [workspace]
-  (let [count (get-in workspace ["workspaceSubmissionStats" "runningSubmissionsCount"])]
-    (cond (not (nil? (get-in workspace ["workspaceSubmissionStats" "lastFailureDate"]))) "Exception"
-          (zero? count) "Complete"
-          :else "Running")))
+  (let [lastSuccess (js/moment (get-in workspace ["workspaceSubmissionStats" "lastSuccessDate"]))
```

Comment:
Clojure naming conventions here? Unless this makes more sense as JavaScript naming. Not sure if that was intentional.